src/main/java/seedu/event/Event.java

```diff
@@ -15,9 +15,9 @@
      * Empty constructor. Sets name as "unnamed"
      */
     public Event() {
-        setName(null);
-        setDatetime(null);
-        setVenue(null);
+        setName("");
+        setDatetime("");
+        setVenue("");
         this.participantList = new ArrayList<>();
```

Comment:
Does it mean that there is always an empty participationList and empty attendanceList?